lib/Subscriber/Core/MappingSubscriber.php

```diff
@@ -274,11 +268,19 @@ private function hydrateReferenceField(
                 $this->getDefaultValue($fieldMapping)
             );
 
-            if ($referencedNode) {
-                $accessor->set(
+            try {
+                if ($referencedNode) {
+                    $accessor->set(
                     $fieldName,
-                    $this->proxyFactory->createProxyForNode($document, $referencedNode, $options)
+                    $proxyFactory->createProxyForNode($document, $referencedNode, $options)
                 );
+                }
+            } catch (\Exception $e) {
```

Comment:
But why not catching the `DocumentManagerException` again? The only throw I can encounter in the above code is a `RuntimeException` inheriting from the `DocumentManagerException`? We already had the same discussion 😕